src/app/components/Billboard/index.tsx

```diff
@@ -0,0 +1,110 @@
+/** @jsx jsx */
+import { useContext, forwardRef } from 'react';
+import { jsx } from '@emotion/react';
+import useViewTracker from '#app/hooks/useViewTracker';
+import useClickTrackerHandler from '#app/hooks/useClickTrackerHandler';
+import { EventTrackingMetadata } from '#app/models/types/eventTracking';
+import idSanitiser from '#app/lib/utilities/idSanitiser';
+import Heading from '../Heading';
+import LiveLabel from '../LiveLabel';
+import MaskedImage from '../MaskedImage';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import styles from './index.styles';
+import Text from '../Text';
+
+interface BillboardProps {
+  heading: string;
+  description?: string;
+  lang: string;
+  link?: string;
+  image: string;
+  eventTrackingData?: EventTrackingMetadata;
+  showLiveLabel: boolean;
+  imageUrl?: string;
+  imageUrlTemplate?: string;
+  imageWidth?: number;
+}
+
+const Banner = forwardRef(
+  (
+    { heading, description, link, image, imageUrl, imageUrlTemplate, imageWidth, eventTrackingData, showLiveLabel, lang }: BillboardProps,
+    viewRef,
+  ) => {
+    const { dir } = useContext(ServiceContext);
+    const isRtl = dir === 'rtl';
+
+    // temporarily invoked to resolve GH errors - remove this when we start using
+    // the service context properly 
+    isRtl;
+
+    const isHeaderImage = !!imageUrl && !!imageUrlTemplate && !!imageWidth;
+    const clickTrackerHandler = useClickTrackerHandler(eventTrackingData);
+
+
+    const id = `billboard-${idSanitiser(heading)}`;
+
+    return (
+      <section role="region" aria-labelledby={id} data-testid={id}>
+        <a href={link} css={styles.clickAreaContainer} onClick={clickTrackerHandler}>
+          <div css={styles.headerContainer} ref={viewRef}>
+            <div css={styles.backgroundContainer}>
+              <div css={styles.backgroundColor} />
+            </div>
+            <div css={styles.contentContainer}>
+              <MaskedImage
+                imageUrl={image}
+                imageUrlTemplate={image}
+                imageWidth={660}
+              />
+              <div css={styles.textContainerWithImage}>
+                <Heading level={2} size="paragon" css={styles.heading} id={id}>
+                {showLiveLabel ? (
+                  <LiveLabel lang={lang} id={id}>
+                    {heading}
+                  </LiveLabel>
```

Comment:
We might not be able to use the `LiveLabel` directly like this, as we need to override the color. We might need to do this (or something similar):

We don't need `lang` as this is handled within the LiveLabel component itself.
We also don't need to set the `id` as it clashes with the ID of the `h2`, and it's not recommended to have 2 components with the same ID on a page.

```suggestion
                  <LiveLabel.Pulse css={styles.livePulse} />
                  <LiveLable.Text css={styles.liveText}>
                    {heading}
                  </LiveLabel.Text>
```

Reply:
We might even need to check whether the `{heading}` text needs to be a child or a sibling of the LiveLabel.Text component - it will depend on what happens when we run an a11y swarm.